infra/cifuzz/cifuzz.py

```diff
@@ -320,24 +323,70 @@ def get_files_covered_by_target(latest_cov_info, target_name,
     return None
 
   # Cases like curl there is /src/curl and /src/curl_fuzzers/ are handled.
-  if not oss_fuzz_project_base.endswith('/'):
-    oss_fuzz_project_base += '/'
+  if not src_in_docker.endswith('/'):
+    src_in_docker += '/'
 
   affected_file_list = []
   for file in coverage_per_file:
-    if not file['filename'].startswith(oss_fuzz_project_base):
+    if not file['filename'].startswith(src_in_docker):
       continue
     if not file['summary']['regions']['count']:
       # Don't consider a file affected if code in it is never executed.
       continue
 
-    relative_path = file['filename'].replace(oss_fuzz_project_base, '')
+    relative_path = file['filename'].replace(src_in_docker, '')
     affected_file_list.append(relative_path)
   if not affected_file_list:
     return None
   return affected_file_list
 
 
+def keep_affected_fuzzers(project_name, out_dir, files_changed, src_in_docker):
```

Comment:
nit: probably better named as "remove_unaffected_fuzzers" to more explicitly indicate what this function is doing (removing).

Reply:
+1